### ISSUE TITLE
Preference to allow selectively disabling snap areas

### ISSUE THREAD

**Issue by Cpuroast** [rxhanson/Rectangle#75]
**Is your feature request related to a problem? Please describe.**
I end up hitting the maximize snap area by accident all the time.

**Describe the solution you'd like**
There should be a preference to selectively disable snap areas.
Like a simple checkbox for each snap area, or it could even be a "defaults write" if a UI preference isn't justified.

**Comment by rxhanson** [comment]
That's fair. I'd only be on board with a "defaults write" solution, which makes this easy enough for newcomers to work on.

**Comment by rxhanson** [comment]
I added this in to https://github.com/rxhanson/Rectangle/releases/tag/v0.20.

As mentioned, I only added the ability for the top snap area to be disabled. The rest can be added in quite easily if there's demand for it in the future.

To disable the top snap area, execute the following command:

`defaults write com.knollsoft.Rectangle ignoredSnapAreas -int 1`

@Cpuroast feel free to close this one out if it solves the issue.
